client/src/components/_common/Section/Section.jsx

```diff
@@ -110,9 +110,6 @@ function Section({
   headerClassName,
   manualContent,
   manualHeader,
-  // manualSidebar,
-  // sidebar,
-  // sidebarClassName,
```

Comment:
Reviewers, please read the **Notes** section of the PR description.

I was keen on these properties, but now I prefer more team feedback.

Basically, add sidebar via `content` or via `sidebar`?

Reply:
> @wesleyboar had written this code and the plan was to have sidebar props in the Section component which would be used to pass in the sidebar component to be used in that Section. 

Hmm that is a good thought. I think it works both ways, it sort of depends on how far we want to take the abstraction for a `<Section>` component, and the benefits of doing so. What were your thoughts @wesleyboar ?

Reply:
Sorry for the delay. I have been in my rabbit hole so deep I discovered a new species of subterranean rabbit.

Reply:
My thoughts are to use `content` to include `<Sidebar>`. I am generally pulling back from abstraction, lately. After seeing a way for a Portal to be built fresh (tup-ui), I've realized the abstraction I introduced CEPv2 was often to escape unmanaged global CSS and consolidate repetitive CSS (cuz my brain is hard-wired to UI code). Given that both `<Section>` and `<Sidebar>` (thanks @shayanaijaz) have achieved that goal, I don't see a pressing reason to go further.